macros/src/builtin_fields_list.rs

```diff
@@ -293,4 +293,9 @@ builtin! {
     #[doc= "return an observable stream of the widget disposed event"]
     disposed_stream: LifecycleSubject,
   }
+
+  DelayDropWidget {
+    #[doc= "the widget which delay_drop_until of true will still be hosted by it's parent event after disposing.it's useful to implement disappear animate "]
```

Comment:
How about:

This widget delay the drop of its child until the field `delay_drop_until` is false, but not affect its `dispose` event emit time. It's useful to ensure the disappear-animate display fully.